### ISSUE TITLE
Lithuanian translation

### ISSUE THREAD

**Issue by chubin** [chubin/wttr.in#634]
To do:

* [ ] Translate messages into Bengali
* [ ] Translate help file into Bengali
* [ ] Fix formatting issues in browser
* [ ] Fix formatting issues in terminal
* [ ] PNG rendering
* [ ] v2.wttr.in translation

The translation is almost done in #630

**Comment by chubin** [comment]
@kainengr Please test, it should work now. It looks like absolutely everything was translated and works. Please test.
Have you added your name to the translators list?

**Comment by kainengr** [comment]
Yes, I've added my name. There's a problem with displaying the translation of longer terms (such as "Partly cloudy") because it's cut after 14 characters. I guess there should be a some wrapping because this happens with some other translations (like Latvian, Russian, Polish).

**Comment by chubin** [comment]
It happens with almost all (all?) translation. We need some generalized approach here.
Wrapping is though an option, but not the best one I think.

The rest looks good?

**Comment by kainengr** [comment]
I've noticed I haven't translated the moon phases. Which file are those in? The rest seems ok.

**Comment by chubin** [comment]
Moon phases where? In /Moon? Here: https://github.com/chubin/pyphoon/blob/master/pyphoon/lib/translations.py
